src/shared/sandboxCommandBase.ts

```diff
@@ -150,6 +150,27 @@ export abstract class SandboxCommandBase<T> extends SfCommand<T> {
       this.updateProgress(results, options.isAsync);
       this.reportResults(results);
     });
+
+    lifecycle.on(SandboxEvents.EVENT_MULTIPLE_SBX_PROCESSES, async (results: SandboxProcessObject[]) => {
+      const resumingProcess = results.shift() as SandboxProcessObject;
+      const sbxProcessIds: string[] = [];
+      const sbxProcessStatuses: string[] = [];
+      results.map((sbxProc) => {
+        sbxProcessIds.push(sbxProc.Id);
+        sbxProcessStatuses.push(sbxProc.Status);
+      });
```

Comment:
[`results` is probably the wrong array because of the previous comment.  so, `others` maybe?]

```suggestion
      const sbxProcessIds = results.map((r) => r.Id)
      const sbxProcessStatuses = results.map((r) => r.Status)    
```

Reply:
The shift is by design so this code is correct, but I see your point about how other listeners would be affected.  I'd prefer to do 1 loop (map) versus 2.